src/java/org/apache/cassandra/repair/autorepair/RepairTokenRangeSplitter.java

```diff
@@ -636,7 +638,7 @@ private Collection<Range<Token>> getTokenRanges(boolean primaryRangeOnly, String
         Collection<Range<Token>> wrappedRanges;
         if (primaryRangeOnly)
         {
-            wrappedRanges = StorageService.instance.getPrimaryRanges(keyspaceName);
+            wrappedRanges = TokenRingUtils.getPrimaryRangesForEndpoint(keyspaceName, FBUtilities.getBroadcastAddressAndPort());
```

Comment:
Note: this is deprecated, you may need to change this back when going back to 4.1 as this new method was added for TCM.

Reply:
Ack